### ISSUE TITLE
Dynamic Blog feed

### ISSUE THREAD

**Issue by dy2288** [BCDevExchange/BCDevExchange-app#148]
<a rel="Delivery" href="https://github.com/BCDevExchange/docs/blob/master/discussion/projectstates.md"><img alt="In production, but maybe in Alpha or Beta. Intended to persist and be supported." style="border-width:0" src="http://bcdevexchange.org/badge/3.svg" title="In production, but maybe in Alpha or Beta. Intended to persist and be supported." /></a>

# Story:

As a BCDevExchange visitor I would like to view all BCDevX related blog posts on the site so that I can stay up to date with the project activities.

# Additional detail:

The list of Blog entries in https://bcdevexchange.org/blog is currently static (updates in the application and requires a build).

We need to make this feed dynamic - consuming from http://blog.data.gov.bc.ca/feed/?category=BCDev

As it stands, the WordPress feed currently only shows the last 10 Blog posts. We'd like to consume all the BCDevX blog posts.

Links:

[Blog page](https://bcdevexchange.org/blog)
 
# Story Originator:
* Daniel Yona

# Success criteria:

* Blogs will automatically appear from http://blog.data.gov.bc.ca/ as long as they are in the "BCDev" category in WordPress

# Approach:
* We will need to get the Blog time to expose access to this data (XML or JSON)
* Once the data has been exposed, we can make modifications to our code in order to consume it.
* We may want to build some caching mechanism into this so that we're not querying the data every time. May be as simple as sending the right headers on the feed from the WordPress side.

# Connect with:
* Loren M
* Daniel Y

**Comment by f-w** [comment]
I think there is a bug in either waffle or github. According to waffle docs:
```
Closing Issues via Commits
Including a keyword and the issue # in your commit message will close that issue once it is merged into your default branch. This will automatically move the issue to the Done column on your Waffle board.

keywords:
close, closes, closed
fix, fixes, fixed
resolve, resolves, resolved
```
As you can see above I committed code, referenced the issue and contained keyword `fixes` but the issue is neither closed nor changed state in waffle board.
Please don't manually close the issue. I am trying to show the problem to waffle.

**Comment by kellihsf** [comment]
Hey @f-w, Kelli from Waffle here. The word "issue" in "fixes issue #148" is what's messing it up. Here's what GitHub says about it: https://help.github.com/articles/closing-issues-via-commit-messages/

Hope that helps!

**Comment by f-w** [comment]
@kellihsf thanks for the suggestion. Works like a charm!
